cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -116,12 +170,13 @@ public Integer call() throws Exception {
             DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
             ApplicationsOperations applicationsOperations = new ApplicationsOperations(cfOperations);
 
-            for (Entry<String, ApplicationBean> applicationEntry : applicationBeans.entrySet()) {
-                String applicationName = applicationEntry.getKey();
-                ApplicationBean applicationBean = applicationEntry.getValue();
-                applicationsOperations.create(applicationName, applicationBean, false);
-                Log.info("App created:", applicationName);
-            }
+            // do so that authorization has taken place. else leads to authorization problems
+            // TODO: find better solution
+            cfOperations.getOrganizationId().block();
+            Flux.fromIterable(applicationBeans.entrySet())
+                    .flatMap(appEntry -> applicationsOperations.create(appEntry.getKey(), appEntry.getValue(), false))
+                    .onErrorContinue(Log::warning)
+                    .blockLast();
 
             return 0;
```

Comment:
We should return 1 if there were any problems during creation

Reply:
try catch should be helpful here, also to wrap possible exceptions